Point the manual .NET 1.1 install using tools instead of mentioning vendors being required (#44395)
dotnet/docs: docs/framework/install/run-net-framework-1-1-apps.md

# Run .NET Framework 1.1 apps on Windows 8, Windows 8.1, Windows 10, or Windows 11

> .NET Framework 1.1 is not supported on the Windows 8, Windows 8.1, Windows Server 2012, Windows Server 2012 R2, Windows 10, or Windows 11 operating systems. In some cases, .NET Framework 1.1 is required for an app to run. In those cases, you can [enable .NET Framework 3.5 through the Add Roles and Features Wizard](./on-windows-11.md#install-net-framework-on-windows-11).

Reviewer: Unfortunately a previous PR sort of messed up the instructions in the win 11 article. It's best to link to the 3.5 on Win 10 article for now. I'll file an issue to track looking at the whole story of 3.5 on modern windows.
```suggestion
.NET Framework 1.1 is not supported on the Windows 8, Windows 8.1, Windows Server 2012, Windows Server 2012 R2, Windows 10, or Windows 11 operating systems. In some cases, .NET Framework 1.1 is required for an app to run. In those cases, refer to the [Install .NET Framework 3.5 on Windows article](./dotnet-35-windows.md).
```
